Graphs/Table/index.js

```diff
@@ -79,6 +79,18 @@ class Table extends AbstractGraph {
         this.checkFontsize();
     }
 
+    static getDerivedStateFromProps(nextProps, state) {
+        const { context, selectedColumns, configuration } = nextProps;
+        const columns = configuration.data.columns || [];
+
+        const { filterColumns, removedColumnsKey } = Table.getStaticColumnByContext(columns, context);
+
+        return {
```

Comment:
This will cause constant re-rendering can we first compare the computed  removedColumns and removedColumnsKey against the state and only return if any changes?